infra/resources/modules/commons/apim.tf

```diff
@@ -84,6 +84,23 @@ module "apim_product_ts_management" {
   policy_xml = file("../modules/commons/api_product/ts_management/_base_policy.xml")
 }
 
+module "apim_product_ts_subscription" {
+  source = "github.com/pagopa/terraform-azurerm-v3//api_management_product?ref=v8.44.0"
+
+  product_id   = "ts-subscription-api"
+  display_name = "TRIAL SYSTEM SUBSCRIPTION API"
+  description  = "Product for Trial System users"
+
+  api_management_name = module.apim.name
+  resource_group_name = module.apim.resource_group_name
+
+  published             = false
+  subscription_required = true
+  approval_required     = false
+
+  policy_xml = file("../modules/commons/api_product/ts_management/_base_policy.xml")
+}
+
```

Comment:
**note**: This resource is coming from PR #193.

I branched from that branch, because I needed the reference to the product subscription in order to create the API.

The dependency will force us to merge the PR #193 before this PR (and when the first PR has been merged, we will not see these changes anymore)